docs/data-sources/directory_role_templates.md

```diff
@@ -0,0 +1,44 @@
+---
+subcategory: "Directory Role Templates"
+---
+
+# Resource: azuread_directory_role_templates
+
+Use this data source to access information about activated directory role templates within Azure Active Directory.
```

Comment:
Does this return only activated templates, or does it return all templates whether activated or not?

Reply:
Removed word, not needed as all templates are returned